features/support/env.rb

```diff
@@ -40,6 +41,8 @@
 
 World(FactoryBot::Syntax::Methods)
 
+STOCKIT_ENABLED = true
```

Comment:
This is already stubbed in `spec/env/support.rb` Do we still need it here? If so, better to load the other file to get all the env vars.

Reply:
yeah. I tried that. But cucumber is reading only the main env i.e. `.env` in our local and whatever the `env` is set in our circleci settings. It is not reading any other files like how the `rspec` does.